### PR TITLE
Mayland: Add Color Annotations

### PR DESCRIPTION
**Pull request by kjellr**
Follows the mold of #1944 to add basic color annotations for Mayland. 

Screen Shot 2020-04-30 at 19 48 47

Still needs: 

- [x] Refinement around contrast. 
- [x] Double checking mobile styles.
- [x] Add featured color palettes.

### CODE REVIEW

File: mayland/inc/wpcom-colors.php [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+	 */
+	// Background-color
+	array( '.has-primary-background-color[class]', 'background-color' ),
+	// Text-color
+	// Needs contrast against `bg`
+	array( '.has-primary-color[class]', 'color', 'bg' ),
+
+), __( 'Link Color' ) );
+
+// Text Color
```
Comment (kjellr) [inline mayland/inc/wpcom-colors.php:116]: All of these Text Color annotations need to be double checked. The terminal command wasn't able to process `010101` for me, so I had to find and enter all of these manually. They don't seem to be working, so I'm sure I made a mistake in here somewhere. 🤔

Comment (kjellr) [reply mayland/inc/wpcom-colors.php:165]: Re-added them, and this went away. Must've been a typo somewhere.

**Comment by ianstewart** [comment]
I can't recall if we've discussed this elsewhere but the editor placeholder text can be hard to read in general. Not sure if Mayland specific …

image

**Comment by ianstewart** [comment]
Other than that note colour annotations appear to work as expected in Mayland. 👍

**Comment by allancole** [comment]
Okay cool. I’ll look into that. May address it in another PR with all the Varia themes.

**Comment by allancole** [comment]
Added a fix for the placeholder colors. It’s pretty straight forward and will work the same for all of the rest of the Varia child-themes. I’ll do all the rest together in a separate PR :-).

LGTM!

**Pull request merged**